packages/block-editor/src/store/selectors.js

```diff
@@ -2151,6 +2153,27 @@ const buildBlockTypeItem =
 			blockType.name,
 			'inserter'
 		);
+		const blockVariations = getBlockVariations( blockType.name, 'block' );
+		// Combine inserter and block variations. Block-scope variations without
+		// inserter scope are searchable via slash commands but hidden from browse.
```

Comment:
> It also modifies the behavior of `"block"` scope in variations so that they are searchable via slash commands.
> 
> This means variations with only 'block' scope (not 'inserter') are marked as `isSearchOnly` and filtered from the inserter browse view, but remain searchable in inserter and slash commands.


This new rule causes unintended behavior: variations that are only block-scoped can be searched with the slash inserter, for example, column block variations.

<img width="530" height="380" alt="Image" src="https://github.com/user-attachments/assets/98a2ae0d-cf82-45b4-b9c2-c64426aad5fe" />